notebooks/267-distil-whisper-asr/267-distil-whisper-asr.ipynb

```diff
@@ -15,7 +15,7 @@
     "\n",
```

Comment:
<blockquote> Quantization is the simplest scenario</blockquote><p> </p><p><a href="https://github.com/openvinotoolkit/openvino_notebooks/pull/1184#discussion_r1334286561" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/pull/1184#discussion_r1334286561</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1439/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Removed